go-controller/pkg/node/gateway_udn.go

```diff
@@ -61,6 +61,13 @@ func (udng *UserDefinedNetworkGateway) AddNetwork() error {
 	if err != nil {
 		return fmt.Errorf("could not add VRF %d for network %s, err: %v", vrfTableId, udng.GetNetworkName(), err)
 	}
+	err = udng.addUDNManagementPortIPs(mplink)
```

Comment:
can you please add a code comment here as well above L64 that this should be done in this specific order because of blah blah?